### PR TITLE
Intro

### PR DESCRIPTION
**Pull request by kasparvonbeelen**
Create intro notebook

### CODE REVIEW

File: Sessions/1a-Intro-to-Python.ipynb [hunk 1]
```diff
@@ -99,17 +209,29 @@
     "    - The programming language (e.g. Python, R, Julia, and so on)\n",
     "    - Relevant packages (e.g. pandas, matplotlib, tidyverse, ggplot)\n",
     "\n",
-    "If you are already familiar with creating Python environments, installing libraries and Jupyter Notebooks you can clone [our GitHub repo](https://github.com/Living-with-machines/dhoxss-text2tech) and use the notebooks on your machine.\n",
     "\n",
-    "### The solution!\n",
+    "## Binder\n",
     "\n",
+    "![](images/mybinder.png)\n",
     "[Binder](https://mybinder.org/) is a service that provides your code and the hardware and software to execute it.\n",
```
Comment (mcollardanuy) [inline Sessions/1a-Intro-to-Python.ipynb:233]: I'd add an empty line after the image, so the text is easier to read (and the same for the other figures).

File: Sessions/1a-Intro-to-Python.ipynb [hunk 2]
```diff
@@ -99,17 +209,29 @@
...
    ]
   },
   {
-   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Local Installation\n",
+    "\n",
+    "Oh this will be fun, but later!\n",
```
Comment (mcollardanuy) [inline Sessions/1a-Intro-to-Python.ipynb]: Fun, yes, in quotation marks ;)

**Review by mcollardanuy — DISMISSED** [review]
Hi @kasparvonbeelen, looks good to me!

File: Sessions/1a-Intro-to-Python.ipynb (hunk 2 above)
Comment (kasparvonbeelen) [reply Sessions/1a-Intro-to-Python.ipynb]: Haha, added quotation marks

File: Sessions/1a-Intro-to-Python.ipynb (hunk 1 above)
Comment (kasparvonbeelen) [reply Sessions/1a-Intro-to-Python.ipynb:233]: Done!

**Review by mcollardanuy — APPROVED** [review]

**Pull request merged**